Channel/TwilioChannel.php

```diff
@@ -36,7 +36,7 @@ public function __construct(
 
     public function supports($recipient)
     {
-        if (is_object($recipient) && $recipient instanceof TwilioRecipientInterface) {
+        if (is_object($recipient) && $recipient instanceof PhoneRecipientInterface) {
```

Comment:
Idem `$recipient instanceof PhoneRecipientInterface;`

Reply:
Same as above, will keep the `is_object` test